pkg/handler/v2/constraint/constraint.go

```diff
@@ -352,6 +352,10 @@ type createConstraintReq struct {
 type constraintBody struct {
 	// Name is the name for the constraint
 	Name string `json:"name"`
+
+	// Namespace is the namespace for the constraint
+	Namespace string `json:"namespace,omitempty"`
```

Comment:
you dont need to add this. Its not used anyway. For the body of the DefaultConstraint you dont need to use the Namespace, as it will automatically take the configured kubermatic namespace.

Reply:
Done